Cargo.toml

```diff
@@ -360,31 +367,39 @@ opt-level = 2 # Speed up CPU-side chroma-upsampling (TODO(#7298): move to GPU)
 # egui has a feature where if you hold down all modifiers keys on your keyboard and hover any UI widget,
 # you will see the backtrace to that widget, and we don't want to break that feature in dev builds.
 
-[profile.dev.build-override]
-debug = true # enable debug symbols for build scripts when building in dev (codegen backtraces!)
```

Comment:
have you tested that we still get callstacks when panicing in a build.rs?

Reply:
Yes we do